app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from models import Actor
```

Comment:
pep8 : D

Reply:
what wrong with pep8? in PyCharm i don't have any problems

Reply:
https://peps.python.org/pep-0008/#imports

Reply:
> https://peps.python.org/pep-0008/#imports

ty, i read